utils/misc/misc.go

```diff
@@ -65,24 +81,29 @@ func GetRudderEventVal(key string, rudderEvent interface{}) (interface{}, bool)
 
 //ParseRudderEventBatch looks for the batch structure inside event
 func ParseRudderEventBatch(eventPayload json.RawMessage) ([]interface{}, bool) {
+	// logger.Debug("=== ParseRudderEventBatch === ")
 	var eventListJSON map[string]interface{}
 	err := json.Unmarshal(eventPayload, &eventListJSON)
 	if err != nil {
+		logger.Debug("==err== ", err)
 		return nil, false
 	}
 	_, ok := eventListJSON["batch"]
 	if !ok {
+		logger.Error("not ok -- 1")
```

Comment:
Remove these logs.

Reply:
Any log that can happen at an event level, should be logger.Debug. Otherwise, we could be bombarded with such logs.

Reply:
Got it
done